KarhooUISDK/CommonUI/UIConstants.swift

```diff
@@ -168,7 +168,7 @@ public struct UIConstants {
             /// 280
             public static let sideMenuWidth: CGFloat = 280
             
-            // TODO: think of better names
```

Comment:
Nice hack to silent Lint :)

Reply:
TBF it's not an important one 😅 . Where it was important or doable on the spot I made the changes in the TODO 😄